src/App.js

```diff
@@ -1,26 +1,24 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
```

Comment:
you have changed this logo.svg file location in this pull request (src/logo.svg → src/images/logo.svg). so, it has to update the path here also. Due to that reason build error happed. could you change this path and push again. 
![Capture](https://user-images.githubusercontent.com/35989708/95702975-835ee380-0c6b-11eb-92b1-0556a74f0763.PNG)

Reply:
OHHHHHHHHHHH, I see. That was my bad. I'll fix it right away and do a  PR. Thank you!